tests/cypress/integration/delete-all-terms.test.js

```diff
@@ -0,0 +1,25 @@
+describe('Command: deleteAllTerms', () => {
+  beforeEach(() => {
+    cy.logout();
```

Comment:
We don't need to log out here. Using `cy.login()` will log the user in regardless of the current user login state.